### ISSUE TITLE
boost::filesystem::complete failed on boost 1.86+ in IO.cpp

### ISSUE THREAD

**Issue by lingerer** [norlab-ulaval/libpointmatcher#589]
```
 		#if BOOST_VERSION >= 105000
 				throw runtime_error(string("Cannot open file ") + boost::filesystem::complete(fullPath).generic_string());
```
complete method has been remove in boost 1.86
`	const string& ext(boost::filesystem::extension(path));`
extendsion method also change to path class

**Comment by ryancu7** [comment]
This issue duplicates #574 and will be fixed in the 1.4.4 release.

**Comment by boxanm** [comment]
Hi, the issue with `boost` is resolved in the latest release. You can find it [here](https://github.com/norlab-ulaval/libpointmatcher/releases/tag/1.4.4)
